README.md

```diff
@@ -1,2 +1,35 @@
-# strike-off-objections-web
+# Strike Off Objections Web
+
+### Overview
+
 Web frontend for submitting objections to a company being struck off the register.
+
+### Requirements
+
+In order to run the service locally you will need the following:
+
+- [NodeJS](https://nodejs.org/en/)
+- [Node Version Manager](https://github.com/nvm-sh/nvm)
```

Comment:
This is just a "nice to have" to essential so i would remove

Reply:
Scaled this down